### PR TITLE
Update README.md to include correct sponsorship

### PR DESCRIPTION
**Pull request by ChrisChurilo**
I am uploading the correct logo for the sponsorship. thank you

### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by ChrisChurilo** [comment]
@Torantulino I updated the readme with the Zilliz logo. I sent you an email as well. thank you!

**Comment by chyezhz** [comment]
PR of zilliz cloud support is here. #2127

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Development of this free, open-source project is made possible by all the <a hre
...
+<a href="https://www.zilliz.com/"><img src="https://assets.zilliz.com/Zilliz_Logo_blue_c22591ccb4.png" height="40px" alt="Zilliz" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://roost.ai"><img src="https://user-images.githubusercontent.com/22963551/233059778-055502b4-fa2e-4334-82b6-9a12252fd9e1.png" height="40px" alt="Roost.AI" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://nuclei.ai/"><img src="https://user-images.githubusercontent.com/22963551/233059785-28ce2549-62de-42c2-925a-c0aa08aef6bd.png" height="40px" alt="NucleiAI" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.algohash.org/"><img src="https://user-images.githubusercontent.com/22963551/233061211-9a9ec216-9fc0-46fd-a124-bf951057207c.png" height="40px" alt="Algohash" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.typingmind.com/?utm_source=autogpt"><img src="https://user-images.githubusercontent.com/22963551/233202971-61e77209-58a0-47d9-9f7e-dd081111437b.png" height="40px" alt="TypingMind" /></a>&nbsp;&nbsp;&nbsp;&nbsp;</p></p>
```
Comment (ntindle) [inline README.md:35]: Was it intentional to delete `influxdata` as a sponsor here?

**Comment by Torantulino** [comment]
Closing this, will resolve over email.
Thank you for your generous sponsorship!

**Pull request closed without merging**